components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/db/PinnedSiteEntity.kt

```diff
@@ -28,4 +31,25 @@ internal data class TopSiteEntity(
 
     @ColumnInfo(name = "created_at")
     var createdAt: Long = System.currentTimeMillis()
-)
+) {
+    internal fun toTopSite(): TopSite {
```

Comment:
Curious why this is an inner function but the one below that does the opposite is an extension function. They function the same functionality, so it doesn't matter. 🙂

Reply:
Mostly copied the same style as `SitePeremissionsEntity.kt`. Assuming no changes needed.